### DISCUSSION TITLE
Profile Readme

### DISCUSSION THREAD

**Discussion by AssassinAguilar** [Ashutosh00710/github-readme-activity-graph#68] [Q&A] [answered]
Why this graph in my Profile Readme shows null contribution graph ? 🤕 Am I doing something wrong ? Or I have to make an account of heroku ?
https://github.com/AssassinAguilar

**Comment by Ashutosh00710** [comment]
No, you don't have to create a Heroku account. I have visited your profile and saw that everything is working fine

**Comment by Kshitij978** [comment]
Hey @AssassinAguilar I checked your profile and it seems to be working fine. Can you attach the screenshot?

**Comment by AssassinAguilar** [comment, marked as answer]
@Ashutosh00710 @Kshitij978 Sorry guys I found the problem myself where I am going wrong and fixed it but forget to tell here that it get solve 😅 Sorry for Disturbing 🙏

**Reply by Ashutosh00710** [reply]
No Problem
